### PR TITLE
Active Records Database Arrangement

### PR DESCRIPTION
**Pull request by Srjnnnn**
- The user can create Users.
- The user can create Posts.
- The user can create Comments on those Posts with the Users that they've created.
- The password must be at least 6 characters.
- The username must be at least 4 characters.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hi Team 👋

Nice work so far creating the associations and validations. 👌

However, let's fix this slight issue:

- [x] Please change the Author table to User.

Kindly fix this issue and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

**Comment by Srjnnnn** [comment]
- The 'Authors' has been changed into Users.
- The migration has been done from the beginning.

**Review by mikenath223 — CHANGES_REQUESTED** [review]
**Hi again Team 👋**

Good job creating a migration to rename the table.

- [x] Kindly fix the associations also. There are still associations pointing to Author.

Please fix and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

**Comment by Srjnnnn** [comment]
- Fix associations.

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hey Team. 

There are still some redundant properties left from the previous author table.

- [ ]  Please fix.

- [ ]  Also perhaps consider renaming the `username` column to the word `name`.
- [ ] In addition, it isn't necessary to have a password on the user table this project is simply about building with active_record. You can consider removing the validations for password on the user model.

You are almost there. 💪

File: db/migrate/20200901081623_create_authors.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class CreateAuthors < ActiveRecord::Migration[6.0]
+  def change
+    create_table :authors do |t|
+      t.string :username
```
Comment (mikenath223) [inline db/migrate/20200901081623_create_authors.rb]: - [ ] rename this column to `name` not `username`

File: db/migrate/20200901082951_create_posts.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+class CreatePosts < ActiveRecord::Migration[6.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :body
+      t.integer :author_id
```
Comment (mikenath223) [inline db/migrate/20200901082951_create_posts.rb]: - [ ] Change this to reflect the new user table as `user_id`, by making a new migration.

File: db/migrate/20200901085205_create_comments.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[6.0]
+  def change
+    create_table :comments do |t|
+      t.string :title
+      t.text :body
+      t.integer :author_id
```
Comment (mikenath223) [inline db/migrate/20200901085205_create_comments.rb]: - [ ] Also this too make a migration to change it to `user_id`.

File: db/migrate/20200901081623_create_authors.rb (hunk 1 above)
Comment (Srjnnnn) [reply db/migrate/20200901081623_create_authors.rb]: The authors.rb has been deleted already! There is no file like that so from where you're looking for we did not understand.

File: db/migrate/20200901082951_create_posts.rb (hunk 2 above)
Comment (Srjnnnn) [reply db/migrate/20200901082951_create_posts.rb]: It's already user_id. I believe you are looking at an old commit.

File: db/migrate/20200901085205_create_comments.rb (hunk 3 above)
Comment (Srjnnnn) [reply db/migrate/20200901085205_create_comments.rb]: It's already user_id. I believe you are looking at an old commit.

**Comment by Srjnnnn** [comment]
- We believe that you've looked at different commit. We've already migrated from the beginning and now there is nothing as author_id.
- The username changed into the name.
- The password has been deleted.

**Review by mosesogwo — APPROVED** [review]
**Hello!:wave: You have done well on this :100:**

You can now merge this pull request.

_**Cheers:stars:**_

### Status: Approved :heavy_check_mark:

**Pull request merged**